src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -96,49 +96,45 @@
 		public static TankDriveShifting chassis;
-		public static MotionController chassisDriveMC;
 		public static BallIO ballIO;
 		public static GearIO gearIO;
+		public static Climber climber;
 		public static Hopper hopper;
-		public static Subsystem[] mainSubsystems;
-		public static CustomPIDController lidarMC;
+		public static CustomXbox driverXbox;
+		public static CustomJoystick operatorStick;
+		public static TeensyController teensyStick;
+		public static FusibleNavX navx;
+		public static AligningCamera alignCamera;
 		public static CANEncoder lidarTurnEncoder;
+		public static CustomPIDController lidarMC;
 		public static LIDAR lidar;
-		public static FusibleNavX navx;
-		public static Climber climber;
+		public static MotionController chassisDriveMC;
 		public static MotionController chassisTurnMC;
-		public static AligningCamera alignCamera;
+		public static Subsystem[] mainSubsystems;
 	}
 
 	public RobotMap() {
+		// Chassis
 		Component.pdp = new PDP();
-		Component.shifter = new SolenoidShifters(Port.Pneumatics.solenoidUp, Port.Pneumatics.solenoidDown);
-		Component.navx = new FusibleNavX(SerialPort.Port.kMXP);
-		Component.chassisDriveMC = new CustomPIDController(0.01, 0.0, -0.02, RobotMap.Component.navx);
-		Component.chassisDriveMC.setInputRange(-180, 180);
-		Component.chassisDriveMC.setContinuous(true);
-		Component.leftWheelEncoder = new CANEncoder("LeftEncoder", Port.CAN.leftEncoder, false);
-		Component.rightWheelEncoder = new CANEncoder("RightEncoder", Port.CAN.rightEncoder, false);
+		Component.leftWheelEncoder = new CANEncoder("LeftEncoder", Port.CAN.leftEncoder);
+		Component.rightWheelEncoder = new CANEncoder("RightEncoder", Port.CAN.rightEncoder);
 		Component.leftWheelEncoder.setDistancePerPulse(Metrics.WHEEL_PULSES_PER_REVOLUTION);
 		Component.rightWheelEncoder.setDistancePerPulse(Metrics.WHEEL_PULSES_PER_REVOLUTION);
-		Component.chassisDriveMC = new CustomPIDController(0.001, 0.0, -0.002,
-			new EncoderGroup(100, Component.leftWheelEncoder, Component.rightWheelEncoder));
-		Component.leftWheel = new Motor("LeftWheel", false, new AccelerationCap(Component.pdp),
+		Component.leftWheel = new Motor("LeftWheel", new AccelerationCap(Component.pdp),
 			new VictorSP(Port.PWM.leftDriveA), new VictorSP(Port.PWM.leftDriveB));
 		Component.leftWheel.setInverted(true);
-		Component.rightWheel = new Motor("RightWheel", false, new AccelerationCap(Component.pdp),
+		Component.rightWheel = new Motor("RightWheel", new AccelerationCap(Component.pdp),
 			new VictorSP(Port.PWM.rightDriveA), new VictorSP(Port.PWM.rightDriveB));
 		Component.rightWheel.setInverted(true);
+		Component.shifter = new SolenoidShifters(Port.Pneumatics.shifterUp, Port.Pneumatics.shifterDown);
+		Component.chassis = new TankDriveShifting("2017-Chassis", Component.leftWheel, Component.rightWheel, Component.shifter);
 		// BallIO
 		Motor ballioDirectionalRoller = new Motor(new CANTalon(Port.CANMotor.ballioDirectionalRoller));
```

Comment:
Should we use the same AccelerationCap for all of them, or reconstruct it for every motor?